translations/en.json

```diff
@@ -172,6 +172,7 @@
 	"copy-transaction-hash": "Copy transaction ID",
 	"cost-fee": "Cost/Fee",
 	"counter-9-plus": "9+",
+	"counter-99-plus": "99+",
```

Comment:
that's the only change required by PR, please revert all changes below in this file